components/directory-size-exporter/README.md

```diff
@@ -0,0 +1,33 @@
+# Directory Size Exporter
+
+## Overview
+
+The directory size exporter is monitoring operators storage by collecting information about the size of top level directories as metrics. Meanwhile, Prometheus will pull the metrics. 
```

Comment:
```suggestion
The directory size exporter is a typical metrics exporter in the prometheus format. It is meant to run as a sidecar to an application to watch a specific application file storage and export metrics about that storage. At the moment it is used only as a sidecar for the `telemetry-fluent-bit` instances to watch the file buffer sizes
```
The first sentences should provide the full context, here a suggestion. I think it is important to outline what it is exporting metrics and usually runs as a sidecar to an application